cmd/tealdbg/debugger_test.go

```diff
@@ -102,7 +102,7 @@ func TestDebuggerSimple(t *testing.T) {
 	debugger.AddAdapter(da)
 
 	ep := logic.NewEvalParams(make([]transactions.SignedTxnWithAD, 1), &proto, nil)
-	ep.Debugger = debugger
+	ep.Debugger = logic.MakeLegacyDebuggerAdaptor(debugger)
```

Comment:
As we have begun to see that we'll need this technique for many features (like _view_ functions, and returning deltas on a txn by txn basis), I think think the main "thing" should be `ep.Tracer` and the interface should be a TxnTracer, or perhaps EvalTracer, rather than a `Debugger`. (Though the right handle side of this assignment woyuld still talk about a Debugger, as it's an adapter from Debugger to Tracer.)